### DISCUSSION TITLE
WARNING The app is not listening on the expected address and will not be reachable by fly-proxy.

### DISCUSSION THREAD

**Discussion by MoSattler** [epicweb-dev/epic-stack#825] [Q&A] [answered]
When deploying a fresh epic-stack app, I get this error message when deploying via Github Actions: 

```
> Waiting for machine 784e109c9d76d8 to reach a good state
> Machine 784e109c9d76d8 reached started state
WARNING The app is not listening on the expected address and will not be reachable by fly-proxy.
You can fix this by configuring your app to listen on the following addresses:
> Running smoke checks on machine 784e109c9d76d8
  - 0.0.0.0:8080
Found these processes inside the machine with open listening sockets:
  PROCESS        | ADDRESSES                             
-----------------*---------------------------------------
  litefs mount   | [::]:20202                            
  /.fly/hallpass | [XXX]:22  
> Running machine checks on machine 784e109c9d76d8
> Checking health of machine 784e109c9d76d8
✔ Machine 784e109c9d76d8 is now in a good state
``` 

I do not know if this is relevant at all, but I wanted to raise it just in case!

**Comment by kentcdodds** [comment, marked as answer]
I'm fairly certain this is happening because fly checks whether the port is bound before the server has started listening. So the only issue is that it takes longer to start than fly expects (which would be cool to optimize, but isn't a real issue).

**Reply by MoSattler** [reply]
Thank you for the clarification!

**Reply by xHomu** [reply]
Noticed this on my own repos recently. Doesn't seem to break anything, so I haven't bothered to figure out what changed.

Thanks for the tip!
